API/hermes_node_api/CMakeLists.txt

```diff
@@ -8,10 +8,10 @@ set(HERMES_ENABLE_EH_RTTI ON)
 add_subdirectory(node_api)
 
 add_library(hermesNodeApi STATIC hermes_node_api.cpp)
-target_link_libraries(hermesNodeApi PRIVATE
-  hermesVMRuntime
-  nodeApi
-  hermesParser)
+target_link_libraries(hermesNodeApi
+  PUBLIC nodeApi
```

Comment:
Since the Node-API headers are included in `hermes_node_api.h` we want `nodeApi` annotated as PUBLIC here.